core/arch/arm/mm/sp_mem.c

```diff
@@ -174,7 +174,7 @@ static void inactivate(struct mobj *mobj)
 	cpu_spin_unlock_xrestore(&sp_mem_lock, exceptions);
 }
 
-const struct mobj_ops mobj_sp_ops __weak __relrodata_unpaged("mobj_sp_ops") = {
+const struct mobj_ops mobj_sp_ops = {
```

Comment:
can be `static` now the weak constraint has been removed (there is a hint to update at line 20 :)

`mobj_ffa_ops` and `sp_ops` should be `static` in their respecitive source file.